src/templates/next/components/shared/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,58 @@
+import { BiLeftArrowAlt, BiRightArrowAlt } from "react-icons/bi"
+import type { PaginationProps } from "../../../types/Pagination"
+
+export const Pagination = ({
+  totalItems,
+  itemsPerPage,
+  currPage,
+  setCurrPage,
+}: Omit<PaginationProps, "type">) => {
+  const totalPages = Math.ceil(totalItems / itemsPerPage)
+
+  return (
+    <nav className="flex flex-row w-full" aria-label="Pagination">
+      {/* Previous button */}
+      <button
+        className="flex flex-row p-1 gap-1 align-middle cursor-pointer disabled:cursor-not-allowed disabled:text-interaction-sub disabled:hover:bg-transparent hover:bg-interaction-main-subtle-hover"
+        aria-label="Previous page"
+        disabled={currPage === 1}
+        onClick={() => {
+          if (currPage !== 1) {
+            setCurrPage(currPage - 1)
+          }
```

Comment:
we might want to strengthen this check, especially if the `currPage` is taken from page params (eg: `www.something.com/test?p=1` - users can manually access `p=0` and bug this)

Reply:
Done in 74a3c42f3bbc2aaed2c3acf63453b9e2b6a05e00.